stackrox/acs-fleet-manager
# Delete dynamic OIDC client within deprovisioning

## Description

Adds the ability to delete previously created OIDC clients using the dynamic client API.

The deletion will only be attempted if no static OIDC client is used.


## Test manual

1. Run fleet manager, fleetshard, operator. Ensure the correct settings are applied for dynamic OIDC client generation.
2. Create a central using i.e. `scripts/create-central.sh`.
3. Verify that an auth provider is created (ensure no log similar to "static config found" is present) and usable.
4. Delete the central instance and verify the OIDC client got deleted successfully.
